.ci/continuous/Jenkinsfile

```diff
@@ -42,6 +55,46 @@ pipeline {
             }
         }
 
+        stage('Snyk Security') {
+            when {
+                expression { gitUtils.isReleaseTag() || gitUtils.isReleaseCandidate() || gitUtils.isReleaseBranch() || gitUtils.isMasterBranch() }
+            }
+            steps {
+                script {
+                    def modulesToScan = ['dummy_contracts', 'dummy_workflows', 'contracts', 'workflows']
```

Comment:
I think we can scrap dummy_contracts and dummy_workflows as looking at CI these are not published

from jenkins log in publish stage only the main contracts and workflow jar are published 

```


[2023-04-07T13:21:49.634Z] Deploying artifact: https://software.r3.com/artifactory/corda-lib-dev/com/r3/corda/lib/reissuance/reissuance-cordapp-contracts/1.0-SNAPSHOT/reissuance-cordapp-contracts-1.0-SNAPSHOT.jar

[2023-04-07T13:21:49.634Z] Deploying artifact: https://software.r3.com/artifactory/corda-lib-dev/com/r3/corda/lib/reissuance/reissuance-cordapp-workflows/1.0-SNAPSHOT/reissuance-cordapp-workflows-1.0-SNAPSHOT.jar

```

i only see 2 Jars published, we only need to scan customer facing artefacts, these modules are likely for testing

Reply:
Yes logic [here ](https://github.com/corda/reissue-cordapp/blob/master/build.gradle#L193-L198) exludes these dummy ones from publishing